test/framework/src/main/java/org/elasticsearch/search/aggregations/bucket/DateHistogramAggregatorTestCase.java

```diff
@@ -6,7 +6,7 @@
  * Side Public License, v 1.
  */
 
-package org.elasticsearch.search.aggregations.bucket.histogram;
+package org.elasticsearch.search.aggregations.bucket;
```

Comment:
This test base class can be moved to `aggregations` module when data histogram has been moved to `aggregations` module as well. For now test framework is convenient because data histogram lives in server and auto date histogram in aggregations module.